src/storage-manager.coffee

```diff
@@ -205,8 +203,19 @@ class _StorageManager
   #
   # Private
 
-  _invalidateCache: (name) ->
-    @collections[name].cache = {}
+  _shouldInvalidateCache: (model) ->
+    return false if !model
+    return false if model.isNew()
```

Comment:
Can these lines be consolidated into

`return false if model?.isNew()`

Reply:
No because it doesn't check for a falsey `model` value.